lib/krane/cluster_resource_discovery.rb

```diff
@@ -34,6 +59,7 @@ def fetch_globals
         fields = full_width_field_names.each_with_object({}) do |name, hash|
           start = cursor
           cursor = start + name.length
+          cursor = 0 if full_width_field_names.last == name.strip
```

Comment:
I can probably solve this in a more elegant manner, but the reason is that the header row doesn't include whitespace after the last field name, even though the last field can be much longer than the fieldname so we were accidentally truncating the last field. This wasn't noticed before because we didn't expose `VERBS` to the public interface.

```
NAME                              SHORTNAMES   APIGROUP                       NAMESPACED   KIND                             VERBS
bindings                                                                      true         Binding                          [create]
```